test/readme.md

```diff
@@ -6,7 +6,7 @@ You can set-up a new test using `yarn new-test` which will start from a template
 
 ## Test Types in Next.js
 
-- e2e: These tests will run against `next dev` and `next start`
+- e2e: These tests will run against `next dev`, `next start`, and deployed to Vercel
```

Comment:
Would it make sense to also run these tests agains “vc dev”?

Reply:
Possibly although since `vc dev` proxies to `next dev` now I'm not sure the specifics we would be testing 🤔